ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-popover.js

```diff
@@ -0,0 +1,243 @@
+import React, { useCallback, useState } from 'react';
+import PropTypes from 'prop-types';
+import FormField from '../../ui/form-field';
+import Box from '../../ui/box';
+import Button from '../../ui/button';
+import Popover from '../../ui/popover';
+import I18nValue from '../../ui/i18n-value';
+import {
+  decGWEIToHexWEI,
+  hexWEIToDecGWEI,
+} from '../../../helpers/utils/conversions.util';
+import {
+  divideCurrencies,
+  multiplyCurrencies,
+} from '../../../../shared/modules/conversion.utils';
+import { TYPOGRAPHY } from '../../../helpers/constants/design-system';
+import { SECONDARY } from '../../../helpers/constants/common';
+import { useGasFeeContext } from '../../../contexts/gasFee';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { useUserPreferencedCurrency } from '../../../hooks/useUserPreferencedCurrency';
+import { useCurrencyDisplay } from '../../../hooks/useCurrencyDisplay';
+import AdvancedGasFeeInputSubtext from './advanced-gas-fee-input-subtext';
+
+const AdvancedGasFeePopover = ({ onClose }) => {
+  const t = useI18nContext();
+  const {
+    maxPriorityFeePerGas,
+    maxFeePerGas,
+    estimatedBaseFee,
+    onManualChange,
+  } = useGasFeeContext();
+
+  const [editingInGwei, setEditingInGwei] = useState(false);
+  const [priorityFee, setPriorityFee] = useState(maxPriorityFeePerGas);
+
+  const estimatedBaseFeeInDecGWEI = hexWEIToDecGWEI(estimatedBaseFee, {
+    numberOfDecimals: 6,
+  });
+  const baseFeeMultiplier = divideCurrencies(
+    decGWEIToHexWEI(maxFeePerGas),
```

Comment:
It looks like the max fee per gas is already in gwei, is there a reason to convert it to wei first considering we already have the estimated base fee in gwei? If it's because estimated base fee is a string, then why not convert it to a BigNumber first?